### ISSUE TITLE
Dead IPs in script

### ISSUE THREAD

**Issue by krattai** [kewlfft/opennic-up#8]
Two lines in the script have hard coded IPs and neither of the IPs resolve at this time

```
anycast="185.121.177.177"

# wait for network up 20s*4
log 'Checking network is up'
fping -q -t 20000 "$anycast"

# find out what the IP address of api.opennicproject.org is, fallback static IP address configured
apihost=api.opennic.org
apiip=$(dnslookup "$anycast" "$apihost")
echo $apiip
apiip=${apiip:-"161.97.219.82"}
log "Using $apiip as API host"
```

The IP 185.121.177.177 is down and 161.97.219.82 is abandoned to a VM host provider.

**Comment by kewlfft** [comment]
I am not using this DNS at the moment, if you can suggest the new IPs or the changes

**Comment by kewlfft** [comment]
fixed with a3b1d11
